packages/backend/src/api/routes/auth.ts

```diff
@@ -0,0 +1,68 @@
+import type { Services } from '@services';
+import { type Request, Router } from 'express';
+import type { ApiRoutes, UserSignInDto, UserSignUpDto } from '@vse-bude/shared';
+import { AuthApiRoutes } from '@vse-bude/shared';
+import { wrap } from '@helpers';
+import { apiPath } from '../../helpers/api';
+import { authMiddleware } from '../../auth/middlewares/auth.middleware';
+import type { SignOutDto } from '../../common/types/auth/sign-out.dto';
+import type { UpdateRefreshTokenDto } from '../../common/types/refresh-token/update.refresh-token.dto';
```

Comment:
use kebab-case for file names and don't add type after dot
 
 just update-refresh-token